src/NzbDrone.Core/SkyhookNotifications/SkyhookNotificationProxy.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using NLog;
+using NzbDrone.Common.Cloud;
+using NzbDrone.Common.EnvironmentInfo;
+using NzbDrone.Common.Http;
+
+namespace NzbDrone.Core.SkyhookNotifications
+{
+    public interface ISkyhookNotificationProxy
+    {
+        List<SkyhookNotification> GetNotifications();
+    }
+
+    public class SkyhookNotificationProxy : ISkyhookNotificationProxy
```

Comment:
Do we want to use Skyhook as a name here? That seems a bit confusing since we generally refer to our TVDB as Skyhook. Totally up for a name for it, but I don't think Skyhook is it.